base/util.go

```diff
@@ -873,6 +873,20 @@ func Crc32cHashString(input []byte) string {
 	return fmt.Sprintf("0x%x", Crc32cHash(input))
 }
 
+func SplitHostPort(hostport string) (string, string, error) {
+	host, port, err := net.SplitHostPort(hostport)
+	if err != nil {
+		return "", "", err
+	}
+
+	// If this is an IPv6 address, we need to rewrap it in []
+	if strings.Contains(host, ":") {
+		host = "[" + host + "]"
```

Comment:
Was this done as string concat over going the `fmt.Sprintf("[%s]", host)` route intentionally?  I feel like I'm more accustomed to seeing `fmt.Sprintf("[%s]", host)` in code.

Reply:
Really just a case of cut and paste from the corresponding gocb function in this case.  I'm not sure if this is down to the reduces allocs associated w/ concat vs. fmt.Sprintf.  Since this isn't a performance sensitive bit of code, though, I'll change to fmt.Sprintf for better consistency in our code.